test/jdk/tools/jpackage/helpers/jdk/jpackage/test/AdditionalLauncher.java

```diff
@@ -242,9 +251,49 @@ private void verifyShortcuts(JPackageCommand cmd) throws IOException {
         }
     }
 
+    private void verifyDescription(JPackageCommand cmd) throws IOException {
+        if (TKit.isWindows()) {
+            String expectedDescription = getRawPropertyValue("description");
+            if (expectedDescription != null) {
+                Path launcherPath = cmd.appLauncherPath(name);
+                Executor exec = Executor.of("powershell",
+                                             "-NoLogo",
+                                             "-NoProfile",
+                                             "-Command",
+                                             "(Get-Item \\\"" +
+                                             launcherPath.toAbsolutePath() +
+                                             "\\\").VersionInfo | select FileDescription");
+                boolean descriptionIsValid = false;
+                List<String> lines = exec.executeAndGetOutput();
+                if (lines != null) {
+                    for (int i = 0; i < lines.size(); i++) {
+                        if (lines.get(i).trim().equals("FileDescription")) {
+                            i += 2; // Skip "---------------" and move to description
+                            descriptionIsValid =
+                                    expectedDescription.equals(lines.get(i).trim());
+                        }
+                    }
```

Comment:
I kept original implementation. Actually, powershell returns 6 lines: empty, FileDescription, ----, Actual Description, empty, empty. I think it is better to make sure FileDescription is present and go from top to bottom in case if output can be something like: empty, ErrorMessage, ---, Actual error message, empty, empty.